lib/pharos/phases/reset_host.rb

```diff
@@ -8,6 +8,10 @@ class ResetHost < Pharos::Phase
       def call
         logger.info { "Removing all traces of Kontena Pharos ..." }
         host_configurer.reset
+        if @config.addons.dig('kontena-lens', 'enabled')
+          data_dir = @config.addons.dig('kontena-lens', 'data_dir').strip
```

Comment:
NoMethodError strip for NilClass? I don't see a data_dir attribute in the schema. Should this be 'kontena-storage'?

Reply:
Yeah, this is wrong.. going to refactor this to addons.